source/boot_security.c

```diff
@@ -1,11 +1,21 @@
 #include "boot_security.h"
+#include "aes.h"
+#include "hal_log.h"
+#define EXAMPLE_AES_KEY_LEN_IN_BITS		256
+#define EXAMPLE_AESKEY_LEN_IN_BYTES		(EXAMPLE_AES_KEY_LEN_IN_BITS / 8)
+#define AES_IV_LENGTH					(16)
 
 bool boot_authenticate_upgrade_package(boot_upgrade_package_t* package)
 {
     return true;
 }
-
 void boot_decrypt_upgrade_package(boot_upgrade_package_t* package)
```

Comment:
Could you please add a function header in Doxygen style? 

Please see our coding style : https://github.com/OpenIoTAcademy/Template-Embedded/wiki/Coding-Style-%5BTR%5D

Please see the section 10. Dosya ve Fonksiyon Headerlari
and add a function header please.